openedx_tagging/core/tagging/fixtures/language_taxonomy.yaml

```diff
@@ -1296,3 +1296,4 @@
     allow_multiple: false
     allow_free_text: false
     visible_to_authors: true
```

Comment:
Yes, sorry I wasn't clear. We can delete the languages, but we still need to create the taxonomy. However, if it's just creating a single object, I would prefer to move that to be a data migration instead of a fixture, to keep things simple.

Reply:
Ah gotcha.. data migration is fine with me.

Reply:
And if we remove the fixture, we can also remove `openedx_tagging/core/tagging/management/commands/`.